release_docs/README.md

```diff
@@ -0,0 +1,102 @@
+# The `release_docs` directory
+
+## Intro
+
+This directory contains instructions for building and using the library as
+well as the HDF5 history files.
+
+## HISTORY files
+
+The `HISTORY` files contain the history of this branch of HDF5. They fall into
+three categories.
+
+### HISTORY-\[VERSION 1\]-\[VERSION 2\].txt
+
+These files are created when we release a new major version and include all
+the changes that were made to the `develop` branch while creating a major release.
+
+### HISTORY-\[VERSION\].txt
+
+This file contains the changes that were made to a maintenance branch since
+it split off from `develop`. It will also be found in the `develop` branch
+when experimental releases have been created.
+
+### RELEASE.txt
+
+This is the changelog for the current version of the library.
+
+For a MAJOR release (or in `develop`) this files lists all the changes since the
+last major version. For a MINOR release (or in a maintenance branch), this file
```

Comment:
There could be some ambiguity to "since the last major version".  Would "since the initial release of the previous major version" be correct and more precise?  Also it should be "this file lists" and I don't think the spell checker will catch the extra 's' in "files".  Probably "last release" below could also be "previous release"?